<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Regular</class>
 <widget class="QWidget" name="Regular">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="background" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>480</width>
     <height>310</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;
border-radius:10px;</string>
   </property>
   <widget class="QLabel" name="regularLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>HarmonyOS Sans SC</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#ccdff8;
color:#666666;
border-radius:10px;
qproperty-iconSize: 32px 32px;</string>
    </property>
    <property name="text">
     <string>正则表达式</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="regularEdit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>HarmonyOS Sans SC</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#f0f3f8;
border-radius:10px;
padding-left:10px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="actionLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>HarmonyOS Sans SC</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#ccdff8;
color:#666666;
border-radius:10px;
qproperty-iconSize: 32px 32px;</string>
    </property>
    <property name="text">
     <string>运行命令</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="actionEdit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>80</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>HarmonyOS Sans SC</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#f0f3f8;
border-radius:10px;
padding-left:10px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="fromLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>HarmonyOS Sans SC</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#ccdff8;
color:#666666;
border-radius:10px;
qproperty-iconSize: 32px 32px;</string>
    </property>
    <property name="text">
     <string>来源</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QRadioButton" name="console">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>120</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>HarmonyOS Sans SC</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#f0f3f8;
border-radius:10px;
padding-left:10px;</string>
    </property>
    <property name="text">
     <string>控制台</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="group">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>120</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>HarmonyOS Sans SC</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#f0f3f8;
border-radius:10px;
padding-left:10px;</string>
    </property>
    <property name="text">
     <string>群聊</string>
    </property>
   </widget>
   <widget class="QLabel" name="permissionLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>HarmonyOS Sans SC</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#ccdff8;
color:#666666;
border-radius:10px;
qproperty-iconSize: 32px 32px;</string>
    </property>
    <property name="text">
     <string>权限</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>160</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>HarmonyOS Sans SC</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#f0f3f8;
border-radius:10px;
padding-left:10px;</string>
    </property>
    <property name="text">
     <string>需要管理员权限</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>HarmonyOS Sans SC</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#ccdff8;
color:#666666;
border-radius:10px;
qproperty-iconSize: 32px 32px;
</string>
    </property>
    <property name="text">
     <string>保存</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancel">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>260</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>HarmonyOS Sans SC</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#ccdff8;
color:#666666;
border-radius:10px;
qproperty-iconSize: 32px 32px;
</string>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>5</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>HarmonyOS Sans SC</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>正则编辑器</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="close">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>0</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Webdings</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	font-family:&quot;Webdings&quot;;
	text-align:top;
	border:none;
	font-size:20px;
	color:#666666;
}
QPushButton:hover{
	color:#3681ff
}

</string>
    </property>
    <property name="text">
     <string>r</string>
    </property>
   </widget>
   <widget class="QPushButton" name="regularTest">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>260</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>HarmonyOS Sans SC</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#ccdff8;
color:#666666;
border-radius:10px;
qproperty-iconSize: 32px 32px;
</string>
    </property>
    <property name="text">
     <string>正则测试</string>
    </property>
   </widget>
   <widget class="QPushButton" name="remove">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>260</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>HarmonyOS Sans SC</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#ccdff8;
color:#666666;
border-radius:10px;
qproperty-iconSize: 32px 32px;
</string>
    </property>
    <property name="text">
     <string>删除</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
